SonarQube.Client.Tests/SonarQubeService_GetRulesAsync.cs

```diff
@@ -135,6 +135,7 @@ public async Task GetRulesAsync_Active_SonarQubeResponse()
             result.Select(r => r.RepositoryKey).Should().Contain(new[] { "csharpsquid", "csharpsquid", "csharpsquid" });
             result.Select(r => r.IsActive).Should().Contain(new[] { true, true, true });
             result.Select(r => r.Severity).Should().ContainInOrder(new[] { SonarQubeIssueSeverity.Major, SonarQubeIssueSeverity.Critical, SonarQubeIssueSeverity.Minor });
+            result.Select(r => r.IssueType).Should().ContainInOrder(new[] { SonarQubeIssueType.Bug, SonarQubeIssueType.Vulnerability, SonarQubeIssueType.CodeSmell });
```

Comment:
The json response in the test already had a type information in it, so I've just had to add the assertion.